llvm/test/CodeGen/AArch64/wineh-catchret-obj-generation.ll

```diff
@@ -0,0 +1,45 @@
+; RUN: llc -mtriple aarch64-unknown-windows-msvc -filetype=obj %s -o %t.o
```

Comment:
Please output asm and use update_llc_test_checks.py.  (This should check that we generate the correct assembler label, instead of `.LBB0_-1`.)

Reply:
done